staging/src/k8s.io/apiserver/pkg/util/flowcontrol/bootstrap/default-objects.go

```diff
@@ -0,0 +1,283 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package bootstrap
+
+import (
+	"math"
+
+	rmv1a1 "k8s.io/api/flowcontrol/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apiserver/pkg/authentication/user"
+)
+
+// The objects that define an apiserver's initial behavior
+var (
+	// InitialPriorityLevelConfigurations lists exempt first, default second
+	InitialPriorityLevelConfigurations = []*rmv1a1.PriorityLevelConfiguration{
+		DefaultPriorityLevelConfigurationExempt,
+		DefaultPriorityLevelConfigurationDefault,
+	}
+	InitialFlowSchemas = []*rmv1a1.FlowSchema{
+		DefaultFlowSchemaExempt,
+		DefaultFlowSchemaDefault,
+	}
+)
+
+// Initial PriorityLevelConfiguration objects
+var (
+	DefaultPriorityLevelConfigurationExempt = pl(
+		rmv1a1.PriorityLevelConfigurationNameExempt,
+		nil)
+
+	DefaultPriorityLevelConfigurationDefault = plq(
+		rmv1a1.PriorityLevelConfigurationNameCatchAll, 100,
```

Comment:
https://github.com/kubernetes/kubernetes/pull/85268#discussion_r351120205

the same question here

Reply:
Subsequent discussions have favored the following.  The suggested configuration does not eschew the catch-all priority level but rather directs a non-trivial fraction of the traffic to it.  That total number of shares is such that 100 represents an appropriate fraction.